yarn.lock

```diff
@@ -1503,10 +1503,10 @@ algoliasearch-helper@3.6.2:
   dependencies:
     events "^1.1.1"
```

Comment:
helper is somewhere manually at 3.6.2 as well, that should be updated :)

Reply:
Good catch, thanks!